ocs_ci/ocs/openshift_ops.py

```diff
@@ -18,7 +18,7 @@ class OCP(object):
 
     """
     def __init__(self):
-
+        config.load_kube_config(config_file=os.getenv('KUBECONFIG'))
```

Comment:
We have this path also available in our config: config.RUN['kubeconfig_location'] but I don't mind to take it also from env var as we expose there at the start of the execution.